mass_mailing_custom_unsubscribe/README.rst

```diff
@@ -1,74 +1,79 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
-====================================================
-Customizable unsubscribe link on mass mailing emails
-====================================================
+==========================================================
+Customizable unsubscription process on mass mailing emails
+==========================================================
 
-With this module you can set a custom unsubscribe link append at bottom of mass
-mailing emails.
+This addon extends the unsubscription form to let you:
 
+- Choose which mailing lists are not cross-unsubscriptable when unsubscribing
+  from a different one.
+- Know why and when a contact as been unsubscribed from a mass mailing.
 
 Configuration
 =============
 
-To configure unsubscribe label go to Setting > Technical > Parameters > System parameters
-and add a 'mass_mailing.unsubscribe.label' parameter with html to set at bottom
-of mass emailing emails. Including '%(url)s' variable where unsubscribe link
+Unsubscription Reasons
+----------------------
 
-For example:
-
-.. code:: html
-
-  <small>You can unsubscribe <a href="%(url)s">here</a></small>
-
-
-Additionally, you can disable this link if you set this parameter to 'False'
-
-If this parameter (mass_mailing.unsubscribe.label) is not set (or set to '')
-default 'Click to unsubscribe' link will appear. This default text is
-translatable via Settings > Translations > Application Terms > Translated terms
+You can customize what reasons will be displayed to your unsubscriptors when
+they are going to unsubscribe. To do it:
 
+#. Go to *Marketing > Configuration > Unsubscription Reasons*.
+#. Create / edit / remove / sort as usual.
+#. If *Details required* is enabled, they will have to fill a text area to
+   continue.
 
 Usage
 =====
 
+Once configured:
+
+#. Go to *Mass Mailing > Mailings > Mass Mailings > Create*.
+#. Edit your mass mailing at wish, but remember to add a snippet from
+   *Footers*, so people have an *Unsubscribe* link.
+#. Send it.
+#. If somebody gets unsubscribed, you will see logs about that under
+   *Mass Mailing > Mailings > Unsubscriptions*.
+
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/205/8.0
-
+   :target: https://runbot.odoo-community.org/runbot/205/10.0
 
 Known issues / Roadmap
 ======================
 
-* This custom html is not translatable, so as a suggestion, you can define
-  the same text in several languages in several lines.
-
-For example:
-
-.. code:: html
-
-  <small>[EN] You can unsubscribe <a href="%(url)s">here</a></small><br/>
-  <small>[ES] Puedes darte de baja <a href="%(url)s">aquí</a></small>
-
+* This module adds a security hash for mass mailing unsubscription URLs, which
```

Comment:
Please add them a comment on code and on README for serving as reminder on v11 migration.